app/controllers/api/v2/template_combinations_controller.rb

```diff
@@ -36,9 +62,20 @@ def destroy
       end
 
       def find_parent_config_template
-        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id])
-        not_found unless @config_template
-        @config_template
+        @config_template = ConfigTemplate.authorized(:view_templates).find(params[:config_template_id]) rescue nil
+      end
+
+      def requested_resource
```

Comment:
Is this necessary any more?  It looks like find_parent_config_template is handled by the nested resource filter, and that the "return not_found" part could be handled by using find_required_nested_obj rather than optional.  Or is there a subtle difference?